BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -5,40 +5,60 @@ public struct AttributedLabel: Element {
 
     public var attributedText: NSAttributedString
     public var numberOfLines: Int = 0
-    /// The scale to which pixel measurements will be rounded. Defaults to `UIScreen.main.scale`.
-    public var roundingScale: CGFloat = UIScreen.main.scale
+    public var isAccessibilityElement = false
+
+    /// An offset that will be applied to the rect used by `drawText(in:)`.
+    ///
+    /// This can be used to adjust the positioning of text within each line's frame, such as adjusting
+    /// the way text is distributed within the line height.
+    public var textRectOffset: CGPoint = .zero
```

Comment:
Would `UIOffset` be more appropriate here?

Reply:
Sure, I didn't know that was a thing!